readme.md

```diff
@@ -21,8 +21,122 @@ text including copyright information.
 Please visit https://www.teeworlds.com/ for up-to-date information about
 the game, including new versions, custom maps and much more.
 
-Instructions to build the game can be found at 
-https://teeworlds.com/?page=docs&wiki=compiling_everything. In
-particular, you will need SDL2 and FreeType installed.
-
 Originally written by Magnus Auvinen.
+
+
+Building on Linux or macOS
+==========================
+
+Installing dependencies
+-----------------------
+
+    # Debian/Ubuntu
+    sudo apt install build-essential cmake git libfreetype6-dev libsdl2-dev libpnglite-dev libwavpack-dev python3
+
+    # Fedora
+    sudo dnf install @development-tools cmake gcc-c++ git freetype-devel mesa-libGLU-devel pnglite-devel python3 SDL2-devel wavpack-devel
+
+    # Arch Linux (doesn't have pnglite in its repositories)
+    sudo pacman -S --needed base-devel cmake freetype2 git glu python sdl2 wavpack
+
+    # macOS
+    sudo brew install cmake sdl2

```

Comment:
Should add freetype too. It's not installed per default.

Reply:
Done.